android/expoview/src/main/java/host/exp/exponent/storage/ExponentSharedPreferences.java

```diff
@@ -56,6 +56,7 @@
   public static final String SAFE_MANIFEST_KEY = "safe_manifest";
   public static final String EXPO_AUTH_SESSION = "expo_auth_session";
   public static final String EXPO_AUTH_SESSION_SECRET_KEY = "sessionSecret";
+  public static final String HAS_CLEARED_LEGACY_OKHTTP_CACHE_KEY = "has_cleared_legacy_okhttp_cache";
```

Comment:
I'm a bit hesitant to put a key called `has_cleared_legacy_okhttp_cache` on devices with newly installed apps that aren't affected at all by this cache transition. A more forward-looking approach would be to have a key like `okhttp_cache_version` with the value `1` (no key = we need to try to clear the cache). Then if we were to make cache changes in the future, we could bump this key from 1 to 2.